src/widget/loginscreen.cpp

```diff
@@ -31,6 +31,16 @@
 #include <QMessageBox>
 #include <QDebug>
 
+#ifdef Q_OS_WIN32
+    #include <windows.h>
+#else
+    #include <X11/XKBlib.h>
```

Comment:
ok, I interpreted "dependency changes" in the literal sense, because it'll break most build scripts. Of course it shouldn't need a vote if ~everyone agrees, but I think it should at least get brought up before merging.